database/seeds/CoordSeeder.php

```diff
@@ -0,0 +1,102 @@
+
+<?php
+
+use Illuminate\Database\Seeder;
+use App\Coords;
+
+
+class CoordSeeder extends Seeder
+{
+
+
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $array = [
```

Comment:
This probably doesn't really matter for seeders, but it might be nice for readability to have a descriptive name for this array. What is this an array of?